packages/loot-core/src/server/budget/goaltemplates.ts

```diff
@@ -1,39 +1,24 @@
 // @ts-strict-ignore
```

Comment:
Its probably best to not consider the older version of this file when reviewing since so much of the old file was removed.

Reply:
👍 The only thing I've seen in here is that there are a few snake_case variables left over - not a major thing but if it's being touched then it wouldn't hurt to standardise to camelCase.

Eg line 13, 20, 30 etc...